metadata-ingestion/tests/integration/lookml/lkml_samples/owners_refinement.view.lkml

```diff
@@ -0,0 +1,6 @@
+view: +owners {
```

Comment:
could you add a comment saying that this file was added to test cross-file refinement resolution?

Reply:
done